src/olympia/pages/urls.py

```diff
@@ -48,7 +48,7 @@
         name='pages.sunbird'),
     url('^sunbird/', lambda r: perma_redirect(reverse('pages.sunbird'))),
 
-    url('^shield-study-1/',
```

Comment:
Good catch. All entry points to that study have been discontinued.  Should I remove the corresponding HTML page in this PR or leave the path in urls.py for a later cleanup?